game/typechart.py

```diff
@@ -26,6 +28,8 @@ def __repr__(self) -> str:
 
 
 class Multiplier(Enum):
+    """Type advantage calculations"""
```

Comment:
The calculations aren't actually performed here, so more accurately, this is the possible factors a type can multiplied against. 
`Enum values for possible type advantage factors`

Reply:
Changed to `Enum values for possible type advantage factors`